apps/condo/domains/user/components/auth/ValidatePhoneForm.tsx

```diff
@@ -177,12 +178,11 @@ export const ValidatePhoneForm = ({ onFinish, onReset, title }): React.ReactElem
                                     values={{
                                         phone: (
                                             <span style={{ whiteSpace: 'nowrap' }}>
-                                                {`${showPhone} `}
+                                                {`${formatPhone(showPhone)} `}
                                                 <Typography.Link
                                                     underline
                                                     style={{ color: 'black' }}
-                                                    onClick={() =>
-                                                        setIsPhoneVisible(!isPhoneVisible)}
+                                                    onClick={handleNumberVisible}
```

Comment:
More straight forward strategy here would be handling only flag value by this callback, and format value according to this flag, using useMemo